src/server/routes/viewsRouter.tsx

```diff
@@ -126,4 +127,25 @@ const viewRouter = async (req: Request, res: Response) => {
   res.status(status).send(responseString);
 };
 
+// TODO: remove as soon as it is no longer needed
+const hotjarId = 2555715; // if we discover that this needs to be updated, we will extract it into an environment variable
+const hotjarTrackingScript = configForClient.environment.shouldReportAnalytics
```

Comment:
Just making sure this is the right environment check we want. I dont understand how enabling GA analytics reporting is link to hotjar.

Reply:
sorry ignore my question above, you are checking if it is a production deployment (i was expecting an expilcit flag for checking production environment). Was just making sure its the right flag to use.

Reply:
Yes; I am checking here that the code is executed in the production deployment. Hotjar is similar to Google Analytics in that both are horrible third-party scripts collecting user data; so both will be enabled in an environment where we need this data collection.

The alternative would have been to create a dedicated environment variable for Hotjar; which I don't think deserves it at this point.